modules/lightning_features/lightning_media/src/MediaHelper.php

```diff
@@ -50,12 +50,7 @@ public function getFileExtensions($check_access = FALSE, array $target_bundles =
     // one that adds an optional second parameter to loadMultiple().
     $storage = $this->entityTypeManager
       ->getStorage('media_bundle');
-    if (!empty($target_bundles)) {
-      $bundles = $storage->loadByProperties(['id' => $target_bundles], $check_access);
-    }
-    else {
-      $bundles = $storage->loadMultiple(NULL, $check_access);
-    }
+    $bundles = $storage->loadMultiple($target_bundles ?: NULL, $check_access);
```

Comment:
@phenaproxima , @balsama -

Only possible to with commit: [d96ffe1](https://github.com/acquia/lightning/pull/370/commits/d96ffe18d72442c942327c60ee6f8d79ae4518fe).